src/detail/detail.js

```diff
@@ -0,0 +1,69 @@
+import { LitElement, html } from "lit-element";
+import styles from "./detail.style.js";
+
+
+class DetailComp extends LitElement {
+
+  static get styles() {
+    return [styles];
+  }
+
+  static get properties() {
+    return {
+      cover: { book_title: Object }
```

Comment:
you can only use `type or `attribute`, it should be like this
```suggestion
      cover: { type: Object }
```

Reply:
Done!